src/neo/SmartContract/Native/Tokens/GasToken.cs

```diff
@@ -19,7 +19,7 @@ internal GasToken()
         internal override void Initialize(ApplicationEngine engine)
         {
             UInt160 account = Blockchain.GetConsensusAddress(Blockchain.StandbyValidators);
-            Mint(engine, account, 30_000_000 * Factor);
+            Mint(engine, account, 100_000_000 * Factor); // Mint for Neo2.x migration. If there is any surplus, we'll burn it later
```

Comment:
100m is the total supply

Reply:
100m is the GAS total supply of Neo2.x. But for Neo3, it's not fixed.

Reply:
50m?

Reply:
Still not enough, the total supply of neo2.x is about 43M, and we also need to consider one or two years for neo2.x, which may produce about 18M GAS.

Reply:
>  we also need to consider one or two years for neo2.x,

During this years, neo3 will earn gas too

Reply:
>  we also need to consider one or two years for neo2.x,

During this years, neo3 will earn gas too

Reply:
We should ensure that the users on Neo2.x will not lost their GAS on Neo3

Reply:
> We should ensure that the users on Neo2.x will not lost their GAS on Neo3

We can ensure this with less than 100m

Reply:
70M ?

Reply:
I think that we can set this value before release neo 3. As @erikzhang said, with 50m would be enough.